network-store-client/src/main/java/com/powsybl/network/store/client/PreloadingNetworkStoreClient.java

```diff
@@ -871,4 +912,16 @@ public void removeConfiguredBuses(UUID networkUuid, int variantNum, List<String>
         ensureCached(ResourceType.CONFIGURED_BUS, networkUuid, variantNum);
         delegate.removeConfiguredBuses(networkUuid, variantNum, configuredBusesId);
     }
+
+    @Override
+    public Optional<ExtensionAttributes> getExtensionAttributes(UUID networkUuid, int variantNum, ResourceType resourceType, String identifiableId, String extensionName) {
+        ensureExtensionAttributesCached(resourceType, networkUuid, variantNum, extensionName);
+        return delegate.getExtensionAttributes(networkUuid, variantNum, resourceType, identifiableId, extensionName);
```

Comment:
a bit weird to call getExtensoinAttributes twice, it works only when the delegate is a CachedNetworkStoreClient .. I see it's the same in the rest of the code, could be cleaned up later

Reply:
The delegate of PreloadingNetworkStoreClient is always a CachedNetworkStoreclient from my understanding (enforced by the constructor): 
https://github.com/powsybl/powsybl-network-store/blob/1d2c57e75f0193c6d358372294460f2872ac1d4d/network-store-client/src/main/java/com/powsybl/network/store/client/PreloadingNetworkStoreClient.java#L60
 or am I missing something ?